app/vendor/mooltipass/app.js

```diff
@@ -92,7 +97,7 @@ mooltipass.app.onMessage = function(senderId, data, callbackFunction) {
         console.log("Cancel request for reqid " + inputObject.reqid);
         
         // Cancel request only implemented in v1.1
-        if (mooltipass.util.getFirmwareFunctionalityVersionFromVersionString(mooltipass.device.version) >= "v1.1" && mooltipass.device.currentReqid == inputObject.reqid)
+        if (mooltipass.util.getFirmwareFunctionalityVersionFromVersionString(mooltipass.device.version?mooltipass.device.version:'0') >= "v1.1" && mooltipass.device.currentReqid == inputObject.reqid)
```

Comment:
are you checking for a null device.version?
Would it maybe be better to set it by default to "v1"?